openapi.yaml

```diff
@@ -740,10 +740,20 @@ paths:
                         example: USD
                       default_plan:
                         type: string
-                        description: >-
-                          Name of the default plan to use when the user doesn't
-                          specify a plan or has no default plan has been assigned
-                          for the user.
+                        description: |-
+                          Name of the plan the back-end uses for billing in case 
+                          1. the user has not subscribed to a specific plan
+                             (see `default_plan` in `GET /me`) and
+                          2. also did not provide a specific plan with the
+                             processing request.
+
+                          If a free plan is available at the back-end, it is 
+                          probably most useful to provide this as the back-end
+                          wide default plan and override it with paid plans through
+                          the user-specific default plan in `GET /me`.
+                          Otherwise, users that have not set up payment yet MAY
+                          receive an error for each processing requests where
+                          they did not provide a free plan specifically.
```

Comment:
As  I noted elsewhere, but probably good to pin this note here:

I'm questioning if it's useful to define a default plan at the global level, if there is already a default plan at the user level. Conceptually that would only be useful if you want to associate a plan to anonymous requests, but what sense does that make?

I understand that removing the default plan from `/`  is technically a breaking change, but practically that's a non-issue as there is no back-end that already handles plans/billing for real.

Reply:
It might not be most useful, but it can simply be omitted for now. Are you aiming at deprecating the global one for 1.x?

Reply:
Indeed, a back-end can choose to not support it. 
The issue is mainly that it makes the spec/documentation confusing IMHO. If the global one is removed, you don't need the three step priority scheme, you just need one straightforward sentence.

Reply:
Yes, but we can't simply remove it for now. Added it to the list of 2.0.0 clean-up tasks: #374 

Also, I could see us moving the whole billing part into the user details as also the currency or billing plans could be user specific.